app/retail/templates/shared/nav.html

```diff
@@ -26,7 +26,7 @@
       <img id="logo" src="{% static "v2/images/logo_large.png" %}" data-hover="{% static "v2/images/logo_med_hover.gif" %}" width="150" height="50" />
     {% endif %}
   </a>
-  {% if not request.path|is_in_list:"/,/contributor,/about,/mission,/vision,/results,/slack,/help,/how/funder,/how/contributor,/tools" %}
+  {% if not request.path|is_in_list:"/,/contributor,/about,/mission,/vision,/not_a_token,/results,/slack,/help,/how/funder,/how/contributor,/tools" %}
```

Comment:
you would have to resolve the conflicts here 😅 
added a regex template tag to ensure this shows up only on relevant pages 
rebase and you'll see the changes :P

Reply:
hahaha I'm looking at it right now 2 new pages more and we will  going end with a 1mb page with that " if not request.path" hahah I will fix it